### DISCUSSION TITLE
error while installing pip to code-server

### DISCUSSION THREAD

**Discussion by Mihailholop** [coder/code-server#6331] [Q&A]
~/storage $ pkg install python-pip
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 lld : Depends: libllvm (= 16.0.6-1) but 16.0.6 is to be installed
E: Unable to correct problems, you have held broken packages

I got a code-server error, everything I was looking for did not help me, tell me what to do

**Comment by code-asher** [comment]
This does not seem to be related to code-server?

Maybe uninstalling libllvm and trying again will help, or maybe your package manager has a way to fix broken packages.

**Reply by Mihailholop** [reply]
how can i fix broken packages i am programming from my phone everything used to work for me after deleting the application i started getting this error when installing python-pip

**Reply by code-asher** [reply]
It depends on the package manager.  Is this Termux?  I am not really sure how to do it but since it says you have held packages maybe something like this will work:

```
pkg update
pkg unhold libllvm
pkg reinstall libllvm
```

If this does not work then you might get better advice in https://github.com/termux/termux-packages
